source/assets/javascripts/_gradient-background.js

```diff
@@ -1,7 +1,33 @@
-$(document).mousemove(function(e){
-  var $width = ($(document).width())/255;
-  var $height = ($(document).height())/255;
-  var $pageX = parseInt(e.pageX / $width,10);
-  var $pageY = parseInt(e.pageY / $height,10);
-  $("body").css("background", "linear-gradient(135deg, rgb(112, "+$pageX+", "+$pageY+"), rgb(255, "+$pageY+", "+$pageX+")" );
+$(function() {
+  var lastXPosition, lastYPosition;
+
+  $(document).mousemove(function(e){
+    if(lastXPosition == e.clientX && lastYPosition == e.clientY) {
+      return
+    }
+
+    lastXPosition = e.clientX
+    lastYPosition = e.clientY
+
+    console.log(e)
+    var $width = ($(document).width())/255;
+    var $height = ($(document).height())/255;
+    var $pageX = parseInt(e.clientX / $width,10);
+    var $pageY = parseInt(e.clientY / $height,10);
```

Comment:
Just out of curiosity how does this work? I'm not familiar with the part `$width,10`. What does the comma mean?